<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Load4VolumesDialog</class>
 <widget class="QDialog" name="Load4VolumesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load 4 Volumes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="volButton1">
     <property name="text">
      <string>Volume 1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QListWidget" name="volList1"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="volButton3">
     <property name="text">
      <string>Volume 3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="volList3"/>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="volList2"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="volButton2">
     <property name="text">
      <string>Volume 2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="volList4"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="volButton4">
     <property name="text">
      <string>Volume 4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Load4VolumesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Load4VolumesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
